app/routes/markdown-to-html.tsx

```diff
@@ -0,0 +1,90 @@
+import Copy from "~/components/copy";
+import { useCallback, useMemo } from "react";
+import Code from "~/components/code";
+import { noop } from "~/common";
+// import { metaHelper } from "~/utils/meta";
+// import { utilities } from "~/utilities";
+import Utiliti from "~/components/utiliti";
+import Box, { BoxContent, BoxTitle } from "~/components/box";
+import { marked } from "marked";
+import DOMPurify from "dompurify";
+
+// export const meta = metaHelper(
```

Comment:
This is not working because its called `markdownToHtml` and not `markdown`. I wonder why typescript isn't helping here, is there a better approach?

Reply:
It was called markdown before I commented that out.  Then I changed it to markdownToHtml to match the naming convention.  But maybe that was the problem that it was called markdown but the file is called markdown-to-html?

Reply:
Yeah I am good with calling it `markdownToHtml` - let me know if your error goes away after your rename it in the utilities file.